mobile/android/base/android_strings.dtd

```diff
@@ -8,19 +8,25 @@
 <!ENTITY firstrun_panel_title_welcome "Witamy">
 <!ENTITY onboard_start_message3 "Przeglądaj z programem &brandShortName;">
 <!ENTITY onboard_start_subtext3 "Spraw, aby przeglądanie mobilne stało się prawdziwie Twoim.">
-<!ENTITY onboard_start_button_account "Zaloguj się do przeglądarki &brandShortName;">
+
+<!ENTITY firstrun_urlbar_message "Witamy w przeglądarce &brandShortName;">
+<!ENTITY firstrun_urlbar_subtext "Wyszukuj szybciej dzięki pomocnym sugestiom.">
+<!ENTITY firstrun_bookmarks_title "Historia">
+<!ENTITY firstrun_bookmarks_message "Ulubione strony, zawsze dostępne">
+<!ENTITY firstrun_bookmarks_subtext "Wyświetlaj wyniki z zakładek i historii podczas wyszukiwania.">
+<!ENTITY firstrun_data_title "Dane">
+<!ENTITY firstrun_data_message "Mniej danych, więcej oszczędności">
+<!ENTITY firstrun_data_subtext "Blokuj obrazy, aby zmniejszyć zużycie danych.">
+<!ENTITY firstrun_sync_title "Sync">
```

Comment:
Personally, I would use "Synchronizacja" here but "Sync" technically (as short brand name) is probably also OK.